README.md

```diff
@@ -2,7 +2,8 @@
 
 Welcome to React Native Gallery. A React Native Windows application which displays the range of React Native components with Windows support. React Native Gallery is currently live in the Microsoft Store! Follow the link to [download](https://www.microsoft.com/en-us/p/react-native-gallery/9npg0b292h4r).
 
-![gallery_welcome](https://user-images.githubusercontent.com/34109996/115446025-777c4d00-a1cb-11eb-8a8f-c1758613c09f.png)
+![gallery_welcome](https://github.com/TatianaKapos/react-native-gallery/assets/42554868/6f78c5fd-3e20-4eaf-aa04-34d6289cfb0c)
```

Comment:
No worse than the link that was there before. But do we make more stable links in the .github folder or something?

Reply:
I just added it to the assets folder!